pkg/services/rpcsrv/subscription_test.go

```diff
@@ -261,6 +268,39 @@ func TestFilteredSubscriptions(t *testing.T) {
 				require.Equal(t, "my_pretty_notification", n)
 			},
 		},
+		"notification matching contract hash and parameter": {
+			params: `["notification_from_execution", {"contract":"` + testContractHash + `", "parameters":[{"type":"Any","value":null},{"type":"Hash160","value":"449fe8fbd4523072f5e3a4dfa17a494c119d4c08"}]}]`,
+			check: func(t *testing.T, resp *neorpc.Notification) {
+				rmap := resp.Payload[0].(map[string]any)
+				require.Equal(t, neorpc.NotificationEventID, resp.Event)
+				c := rmap["contract"].(string)
+				require.Equal(t, "0x"+testContractHash, c)
+				// it should be exact unique "Init" call sending all the tokens to the contract itself
+				parameters := rmap["state"].(map[string]any)["value"].([]any)
+				require.Len(t, parameters, 3)
+				// sender
+				transferReceiverType := parameters[1].(map[string]any)["type"].(string)
+				require.Equal(t, smartcontract.Hash160Type.ConvertToStackitemType().String(), transferReceiverType)
+				transferReceiver := parameters[1].(map[string]any)["value"].(string)
+				hashExp, err := hex.DecodeString(testContractHash)
+				require.NoError(t, err)
+				slices.Reverse(hashExp)
+				hashGot, err := base64.StdEncoding.DecodeString(transferReceiver)
+				require.NoError(t, err)
+				require.Equal(t, hashExp, hashGot)
```

Comment:
1. In a separate commit make: s/testContractHash/testContractHashLE
2. Declare `var testContractHash, _ = util.Uint160DecodeStringLE(testContractHashLE)` similar to `nfsoHash` declaration.
3. Replace selected code with `require.Equal(t, base64.StdEncoding.EncodeToString(testContractHash.StringBE(), transferReceiver)`. The shorter the better.